pkg/printers/internalversion/printers.go

```diff
@@ -283,6 +283,7 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Status", Type: "string", Description: apiv1.PersistentVolumeStatus{}.SwaggerDoc()["phase"]},
 		{Name: "Claim", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["claimRef"]},
 		{Name: "StorageClass", Type: "string", Description: "StorageClass of the pv"},
+		{Name: "VolumeMode", Type: "string", Description: apiv1.PersistentVolumeSpec{}.SwaggerDoc()["volumeMode"]},
```

Comment:
including volume mode seems fine, but should it be set as `Priority: 1` so it only appears in wide mode? there are a *lot* of columns here already... how wide is a normal PV when printed?

Reply:
Currently using `kubectl get pv/pvc` or `kubectl get pv/pvc -o wide` would get the same results. I not sure if it should be special for the column `volumeMode`

Reply:
as we exceed the number of columns that prints in an average terminal, we have to split into primary/secondary columns

Reply:
Done!